.github/workflows/build.yml

```diff
@@ -18,23 +19,20 @@ jobs:
         toolchain: ${{ matrix.rust-toolchain }}
         components: rust-src, clippy, rustfmt
         targets: x86_64-unknown-none, riscv64gc-unknown-none-elf, aarch64-unknown-none, aarch64-unknown-none-softfloat
+    - uses: Swatinem/rust-cache@v2
     - name: Check rust version
       run: rustc --version --verbose
+    - name: Check code format
+      if: ${{ matrix.arch == 'x86_64' }}
+      continue-on-error: ${{ matrix.rust-toolchain == 'nightly' }}
```

Comment:
why do we still need these `continue-on-error` after the update of rust toolchain

Reply:
To make CI happy for future nightly updates.